packages/fd/manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Fast Downward",
+    "shortname": "fd",
```

Comment:
Is the shortname necessary? You could use the name of the directory instead. That also makes accessing the package from the script easier and automatically prevents naming conflicts.

Reply:
Makes sense -- I'll just use the directory name. That'll ensure package naming consistency just from the directory structure.